velox/common/file/File.cpp

```diff
@@ -75,20 +75,28 @@ LocalReadFile::LocalReadFile(std::string_view path) {
       "open failure in LocalReadFile constructor, {} {} {}.",
       fd_,
       path,
-      strerror(errno));
+      folly::errnoStr(errno));
   const off_t rc = lseek(fd_, 0, SEEK_END);
   VELOX_CHECK_GE(
       rc,
       0,
       "fseek failure in LocalReadFile constructor, {} {} {}.",
       rc,
       path,
-      strerror(errno));
+      folly::errnoStr(errno));
```

Comment:
So, how is the folly::errnoStr() better than strerror()?

Reply:
`strerror` possibly modifies a global string thus not thread-safe